install-deps.sh

```diff
@@ -364,6 +366,7 @@ else
         case "$ID" in
             fedora)
                 $SUDO dnf install -y dnf-utils
+                $SUDO dnf install -y golang-github-google-jsonnet
```

Comment:
Let's add this dependency to the ceph.spec.in, in the BuildRequires instead. If it has different names for Fedora and CentOS you can use some macros the check whether it's Fedora or CentOS.

Reply:
Done